### ISSUE TITLE
Facilitate placeholder texts

### ISSUE THREAD

**Issue by cmb69** [cmb69/advancedform_xh#1]
It should be possible to define placeholder texts for the form fields.

Maybe a [workaround via the template system](http://cmsimpleforum.com/viewtopic.php?f=12&t=8202) is sufficient, for the time being.

**Comment by olape-git** [comment]
It would be a good idea to enable the possibility of placeholders for form fields.

**Comment by cmb69** [comment]
@olape-git, I fully agree, but the code is so bad, that I don't dare to add this feature, unless it is refactored to something more malleable – and that will take some time. Sorry.
